bin/test_travis.sh

```diff
@@ -56,7 +56,11 @@ if [[ "${TEST_DOCTESTS}" == "true" ]]; then
     cat << EOF | python -We:invalid
 print('Testing DOCTESTS')
 import sympy
-if not sympy.doctest():
+blacklist = [
+    'sympy/parsing/latex/_antlr/latexparser.py',
```

Comment:
This should be added to the blacklist in sympy/utilities/runtests.py

Reply:
added, but then i suppose you mean removed from here as well

Reply:
Yes. There shouldn't be special things like this in the Travis configuration. We want the tests to pass if someone runs `./bin/doctest` on their own system.